test/browser/integration/prefixed.js

```diff
@@ -0,0 +1,66 @@
+describe('prefixed()', function() {
+
+  // Generic control function used for prefixed() and prefixedCSS() tests
+  // https://gist.github.com/523692
+  function gimmePrefix(prop, obj){
+    var prefixes = ['Moz','Khtml','Webkit','O','ms'],
+      domPrefixes = ['moz','khtml','webkit','o','ms'],
+      elem     = document.createElement('div'),
+      upper    = prop.charAt(0).toUpperCase() + prop.slice(1),
+      len;
+
+    if(!obj) {
+      if (prop in elem.style)
```

Comment:
yeah, most definitely. almost all of `integration` was copied over fairly blindly.
I'll clean up what I see, and after this merges I will work on getting some jscs chocolate all up in our peanut butter